### ISSUE TITLE
Trash in completion if vboxdrv kernel module is not loaded

### ISSUE THREAD

**Issue by dancn** [dougm/vagrant-tramp#16]
The output of VBoxManage is not parsed correctly when vboxdrv kernel
module is not loaded at:

```
  for vm_running in $(VBoxManage list runningvms [...]
```

Unfortunately VBoxManage is not script friendly.
Errors are printed on stdin and the exitcode does not reflect the error:

```
 annotate-output +'' VBoxManage list runningvms
 I: Started VBoxManage list runningvms
 O: WARNING: The vboxdrv kernel module is not loaded. Either there is no module
 O:          available for the current kernel (4.1.0ors+) or it failed to
 O:          load. Please recompile the kernel module and install it by
 O: 
 O:            sudo /etc/init.d/vboxdrv setup
 O: 
 O:          You will not be able to start VMs until this problem is fixed.
 I: Finished with exitcode 0
```

```
VBoxManage --version
10:33:28 O: 4.3.28r100309
```

Maybe is better to check if vboxdrv is loaded before?

**Comment by dougm** [comment]
This should be addressed when we get the changes for PR #15 merged.

**Comment by dougm** [comment]
Fixed by #17
